depends/packages/zeromq.mk

```diff
@@ -19,6 +19,7 @@ define $(package)_set_vars
   $(package)_config_opts += -DENABLE_DRAFTS=OFF -DZMQ_BUILD_TESTS=OFF
   $(package)_cxxflags += -ffile-prefix-map=$($(package)_extract_dir)=/usr
   $(package)_config_opts_mingw32 += -DZMQ_WIN32_WINNT=0x0601 -DZMQ_HAVE_IPC=OFF
+  $(package)_config_opts_mingw32 += -DCMAKE_RC_COMPILER=x86_64-w64-mingw32-windres
```

Comment:
> fix mingw32 build

why bitcoin doesn't need this option? maybe we have some missing package installed in our Dockerfile?

Reply:
I think we have all the packages needed because simply specifying an option is enough to fix it. It's some cmake (or maybe depends in general) related backport that is missing imo but I couldn't figure out which one.

Reply:
should we have TODO here to remove it? Otherwise it may stay forever (or almost forever)

Reply:
I'm fine either way. Having this option here shouldn't be an issue though 🤷‍♂️